src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
```

Comment:
- [ ] It would be better if we use `configureStore` to create our reducer. We can achieve it by first importing it from redux/tookit
```
import { configureStore } from '@reduxjs/toolkit'
const store = configureStore({ reducer: rootReducer })
```
As seen in the requirements below
![Screenshot from 2022-11-08 14-03-06](https://user-images.githubusercontent.com/46388113/200585315-53c7c8ee-90a8-4b68-8a79-72cf6e62ff4c.png)

Reply:
Hello @addod19 please check the instructions again. The day one task does not have this instructions, as am working on this with the next review